package.json

```diff
@@ -4,34 +4,40 @@
   "description": "The Next Generation of CSS-in-JS for Nuxt.js",
   "repository": "nuxt-community/emotion-module",
   "license": "MIT",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "type": "module",
+  "exports": {
+    ".": {
+      "import": "./dist/module.mjs",
+      "require": "./dist/module.cjs"
+    }
+  },
+  "main": "./dist/module.cjs",
+  "types": "./dist/types.d.ts",
   "files": [
-    "dist",
-    "templates"
+    "dist"
   ],
   "scripts": {
-    "build": "siroc build",
-    "dev": "nuxt example",
+    "prepack": "nuxt-module-build",
+    "dev": "nuxi dev playground",
+    "dev:build": "nuxi build playground",
+    "dev:prepare": "nuxt-module-build --stub && nuxi prepare playground",
     "lint": "eslint --ext .js,.vue,.ts .",
-    "release": "yarn build && yarn test && standard-version && git push --follow-tags && npm publish",
-    "test": "yarn lint && jest"
+    "test": "vitest"
   },
   "dependencies": {
-    "@emotion/babel-plugin": "^11.1.2",
-    "@emotion/css": "^11.0.0",
-    "@emotion/server": "^11.0.0"
+    "@emotion/css": "^11.10.5",
+    "@emotion/server": "^11.10.0",
+    "@nuxt/kit": "^3.0.0"
   },
   "devDependencies": {
-    "@babel/preset-env": "latest",
-    "@babel/preset-typescript": "latest",
-    "@nuxt/test-utils": "latest",
-    "@nuxtjs/eslint-config-typescript": "latest",
-    "eslint": "latest",
-    "jest": "latest",
-    "nuxt": "latest",
-    "siroc": "latest",
-    "standard-version": "latest"
+    "@nuxt/module-builder": "^0.2.1",
+    "@nuxt/schema": "^3.0.0",
+    "@nuxt/test-utils": "^3.0.0",
+    "@nuxtjs/eslint-config-typescript": "^12.0.0",
+    "eslint": "^8.31.0",
+    "nitropack": "^1.0.0",
```

Comment:
Do we need nitro as a dev dependency?

Reply:
I think this will be removed. Thanks. I needed it only for development